openreview/conference/invitation.py

```diff
@@ -830,7 +830,7 @@ def __init__(self, conference, note):
             writers = [conference.id],
             signatures = [conference.id],
             invitees = invitees,
-            noninvitees = [conference.get_authors_id(number = note.number)],
+            noninvitees = [conference.get_authors_id(number = note.number), conference.id + '/Paper'+ str(note.number) + '/Secondary_Area_Chair'],
```

Comment:
Just checking, should this be:  `Secondary_Area_Chairs` ?

Reply:
it is only one seconday area chair, why this should be plural?

Reply:
Well, it is kind of a convention that group ids are plural even if there's only one member

Reply:
This group is similar to `AnonReviewer1` or `Area_Chair1`. I don't see a reason to use plural name. 

BTW this group is member of `Area_Chairs` group. 

I think it should be called `Area_Chair2` following the convention of the reviewers names.